tests/components/ecobee/test_climate.py

```diff
@@ -335,3 +389,18 @@ async def test_set_fan_mode_auto(thermostat, data):
     data.ecobee.set_fan_mode.assert_has_calls(
         [mock.call(1, "auto", "nextTransition", holdHours=None)]
     )
+
+
+async def test_turn_aux_heat_on(thermostat, data):
+    """Test when aux heat is set on.  This must change the HVAC mode."""
+    data.reset_mock()
```

Comment:
```suggestion
```

Is this needed.  Does the mock not start in a reset state?

Reply:
I would assume - yes. These two bottom test were the only test I did not change. Removing the reset mock still passes the test